gradle/validation/dependency-check.gradle

```diff
@@ -0,0 +1,12 @@
+// This adds OWASP vulnerability validation of project dependencies
+
+// This should be false only for debugging.
+def failOnError = true
+
+dependencyCheck {
+  autoUpdate=false
```

Comment:
Why not `autoUpdate=true`? I think this is for auto updating the definitions.

From https://jeremylong.github.io/DependencyCheck/dependency-check-gradle/configuration.html

```
Sets whether auto-updating of the NVD CVE/CPE data is enabled. It is not recommended that this be turned to false.
```